features/step_definitions/portfolio.rb

```diff
@@ -1,9 +1,8 @@
 require 'capybara/cucumber'
-require 'capybara/cuprite'
+require 'selenium/webdriver'
```

Comment:
Please don't change the development environment. It makes tests on the continuous integration server to fail.